packages/web/src/pages/edit-page/EditTrackPage.tsx

```diff
@@ -61,7 +61,11 @@ export const EditTrackPage = (props: EditPageProps) => {
 
   const onSubmit = (formValues: TrackEditFormValues) => {
     const metadata = { ...formValues.trackMetadatas[0] }
-    if (!metadata.artwork?.file) {
+    if (
+      metadata.artwork &&
+      'file' in metadata.artwork &&
+      !metadata.artwork?.file
+    ) {
```

Comment:
Okay so specifically the case is that metadata.artwork is defined, and it has a `file` key, but said key has no value?
Was that logical change intended here? Just trying to piece together what's up with this overall change to the `artwork` field logic in various places since I'm not familiar with it.

Reply:
this 'file' in is helping disambiguate it from being a artwork from the backend versus this weird TrackMetadataUpload situation we have going on here. 

the true fix is what you called out above of separating out a "editing X" type from the backend stuff, pretty much entirely